homeassistant/components/mqtt/quality_scale.yaml

```diff
@@ -0,0 +1,131 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      Service actions are set up in `async_setup_entry` and are only set up if the MQTT broker was set up correctly.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions:
+    status: done
+    comment: |
+      The integration can be removed by removing the config entry.
+  entity-event-setup:
+    status: exempt
+    comment: >
+      Entities are updated through dispatchers, and these are
+      cleaned up when the integration unloads.
+  entity-unique-id:
+    status: exempt
+    comment: >
+      This is user configurable, but not required.
+      It is required though when a user wants to use device based discovery.
+  has-entity-name: done
+  runtime-data:
+    status: exempt
+    comment: >
+      Runtime data is not used, as the mqtt entry data is only used to set up the
+      MQTT broker, this happens during integration setup,
+      and only one config entry is allowed.
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  config-entry-unloading: done
+  log-when-unavailable: done
+  entity-unavailable:
+    status: done
+    comment: |
+      Only supported for entities the user has assigned a unique_id.
+  action-exceptions: done
+  reauthentication-flow: done
+  parallel-updates:
+    status: done
+    comment: |
+      No data coordinator is used. Parallel updates are supported through dispatchers.
```

Comment:
We don't have to mention that there is no coordinator used. But do we have the PARALLEL_UPDATES defined?

Reply:
It's also used in actions though. And if there is no limit, it would be nice to have that explicit

Reply:
How? set PARALLEL_UPDATES to `0`?

Reply:
See #131174